test_conformance/api/test_kernel_attributes.cpp

```diff
@@ -0,0 +1,343 @@
+//
+// Copyright (c) 2020 The Khronos Group Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include <iostream>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include "procs.h"
+#include "harness/errorHelpers.h"
+#include "harness/typeWrappers.h"
+#include "harness/kernelHelpers.h"
+#include "harness/parseParameters.h"
+
+#define __CL_ENABLE_EXCEPTIONS
```

Comment:
I think this should be removed.

Reply:
Ok.
Keeping parseParameters.h as it is needed for determining the compilation mode.

Reply:
I was just talking the unused macro definition (GitHub added the extra context automatically). The `kernelHelpers.h` include could be left in for IWYU reasons since there are functions from it used here, but I'm not going to make you change this again before merging.